build-system/tasks/closure-compile.js

```diff
@@ -18,6 +18,7 @@
 const fs = require('fs-extra');
 const argv = require('minimist')(process.argv.slice(2));
 const closureCompiler = require('gulp-closure-compiler');
+const colors = require('ansi-colors');
```

Comment:
Is anything using this?

Reply:
colors.red in several places in this file.